### PR TITLE
Added some notes about macOS in the installation instructions for LLV…

### PR DESCRIPTION
**Pull request by ccadar**
…M 6.0.  SOme other small improvements.

### CODE REVIEW

File: build-stp.md [hunk 1]
```diff
@@ -7,13 +7,19 @@ slug: build-stp
 
 STP is the recommended solver in KLEE.  The instructions below are for release 2.3.3. If you would like to use the upstream version, do not perform the checkout command `git checkout tags/2.3.3` in the instructions below.
 
-STP has a few external dependencies that are listed here as an install command for Ubuntu 18.04:  
+STP has the a few external dependencies that are first listed here as an install command for Ubuntu 18.04:
```
Comment (251) [inline build-stp.md]: `the a`

File: build-llvm60.md [hunk 2]
```diff
@@ -7,26 +7,49 @@ slug: build-llvm60
 
 {% include version_warning.md %}
 
-The current procedure for building KLEE with LLVM 6.0 (recommended) is outlined below. 
+The current procedure for building KLEE with LLVM 6.0 (recommended) is
```
Comment (251) [inline build-llvm60.md]: Your editor might be misconfigured: it introduces annoying line breaks in the middle of sentences (see below).

**Comment by ccadar** [comment]
Thanks, @251 , I fixed those issues, does it look ok to you now?

**Comment by 251** [comment]
@ccadar Yes.

**Comment by ccadar** [comment]
Thanks, merged

**Pull request merged**